enterprise/frontend/src/embedding-sdk/css/index.module.css

```diff
@@ -0,0 +1,12 @@
+@layer reset, mantine, main;
+
+@import "metabase/css/components/form.module.css" layer(main);
+@import "metabase/css/components/list.module.css" layer(main);
+@import "metabase/css/components/modal.module.css" layer(main);
+@import "metabase/css/components/tippy.module.css" layer(main);
+@import "metabase/css/components/grabber.module.css" layer(main);
+@import "metabase/css/admin.module.css" layer(main);
+@import "metabase/css/dashboard.module.css" layer(main);
+@import "metabase/css/query_builder.module.css" layer(main);
+@import "metabase/css/core/animation.module.css" layer(main);
+@import "metabase/css/reset.module.css" layer(reset);
```

Comment:
At least right now to be safe, i don't apply layers for the main app.
So to have layers for the SDK only, a separate entry point for styles is added.